### PR TITLE
Ankit | Test [Electron]  | PROD - Switch icon is not showing proper in the voucher page

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cw0zpae
https://app.clickup.com/t/86cw101tw
https://app.clickup.com/t/86cw0zj4z

### CODE REVIEW

File: apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h4 class="font-15">{{ localeData?.page_heading }}</h4>
                         </ng-container>
                         <ng-template #tagsListItem>
                             <mat-list-item role="listitem" [ngClass]="{ 'active-link': isTagMenuOpened }">
-                                <a href="javascript:;" aria-label="icon" class="d-block clearfix" [matMenuTriggerFor]="tagsMenu" #tagsMenuTrigger="matMenuTrigger" (menuOpened)="isTagMenuOpened = true" (menuClosed)="isTagMenuOpened = false">
+                                <a href="javascript:;" aria-label="icon" class="d-block clearfix" [matMenuTriggerFor]="tagsMenu" #tagsMenuTrigger="matMenuTrigger" (menuOpened)="tagsMenuOpen(true)" (menuClosed)="tagsMenuOpen(false)">
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.html]:
```suggestion
                                <a href="javascript:;" aria-label="icon" class="d-block clearfix" [matMenuTriggerFor]="tagsMenu" #tagsMenuTrigger="matMenuTrigger" (menuOpened)="toggleTagsMenu(true)" (menuClosed)="toggleTagsMenu(false)">
```

File: apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.ts [hunk 2]
```diff
@@ -178,4 +178,22 @@ export class AsideSettingComponent implements OnInit, OnDestroy {
...
         }
     }
+
+    /**
+     * This will show/hide tag manu
+     *
+     * @param {boolean} open
+     * @memberof AsideSettingComponent
+     */
+    public tagsMenuOpen(open : boolean){
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.ts]:
```suggestion
    public tagsMenuOpen(open : boolean): void {
```

File: apps/web-giddh/src/app/vouchers/create/create.component.html [hunk 3]
```diff
@@ -730,7 +730,7 @@ <h2 class="d-inline-block">
                                             *ngIf="currencySwitched">{{ company?.baseCurrency
                                             }}</ng-container><ng-container *ngIf="!currencySwitched">{{
                                             account?.baseCurrency }}</ng-container></span>
-                                    <img alt="switch-icon" height="15px" src="/assets/images/switch-icon.svg"
+                                    <img alt="switch-icon" height="15px" src="{{ imgPath }}images/switch-icon.svg"
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/vouchers/create/create.component.html]:
```suggestion
                                    <img alt="switch-icon" height="15px" src="{{ imgPath }}switch-icon.svg"
```

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/vouchers/create/create.component.html (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vouchers/create/create.component.html]: done

File: apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.ts]: done

File: apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.html (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.html]: done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**